git_machete/cmd.py

```diff
@@ -10,147 +10,23 @@
 import os
 import re
 import shutil
-import subprocess
 import sys
 import textwrap
 
+from git_machete import utils
+from git_machete.utils import RED, YELLOW, ORANGE, GREEN, DIM, BOLD, ENDC, dim, pretty_choices, bold, colored, debug, fmt, underline
+from git_machete.contexts import CommandLineContext
```

Comment:
Hmm I don't like the `contexts` subpackage name, it's extremely non-specific... lemme think of better alternatives...